proxy/envoy/ingress.go

```diff
@@ -169,8 +169,9 @@ func (w *ingressWatcher) generateConfig() (*Config, error) {
 // buildIngressRoute translates an ingress rule to an Envoy route
 func buildIngressRoute(rule *config.RouteRule) *Route {
 	route := &Route{
-		Path:   "",
-		Prefix: "/",
+		Path:        "",
+		Prefix:      "/",
```

Comment:
This is merely an explicit initialization for these values, the code below takes cares of assigning these fields according to the match clause. Ultimately, the `omitempty` kicks in and makes sure only one of them gets marshaled.